applications/nrf5340_audio/src/modules/hw_codec.c

```diff
@@ -123,6 +123,7 @@ static void volume_msg_sub_thread(void)
 /**
  * @brief Write to multiple registers in CS47L63.
  */
+
```

Comment:
No changes to this file should be done in this PR?

Reply:
Replied to this on the comment in the header file